IdentityCore/tests/integration/ios/MSIDKeychainTokenCacheIntegrationTests.m

```diff
@@ -206,7 +206,8 @@ - (void)testRemoveItemWithKey_whenKeyIsValidWithType_shouldRemoveItem
     MSIDKeychainTokenCache *keychainTokenCache = [MSIDKeychainTokenCache new];
     MSIDTokenCacheItem *token = [MSIDTokenCacheItem new];
     token.accessToken = @"some token";
-    MSIDTokenCacheKey *key = [[MSIDTokenCacheKey alloc] initWithAccount:@"test_account" service:@"test_service" generic:self.generic type:nil];
+    token.tokenType = MSIDTokenTypeAccessToken;
+    MSIDTokenCacheKey *key = [[MSIDTokenCacheKey alloc] initWithAccount:@"test_account" service:@"test_service" generic:self.generic type:@(MSIDTokenTypeAccessToken)];
```

Comment:
This unit test function was exactly the same as `testRemoveItemWithKey_whenKeyIsValid_shouldRemoveItem`, so I assume it was copy-pasted but forgotten to modify :)